gradle.properties

```diff
@@ -14,3 +14,4 @@ org.gradle.unsafe.configuration-cache=true
 voice.signing.play=false
 kotlin.daemon.useFallbackStrategy=false
 org.gradle.kotlin.dsl.allWarningsAsErrors=true
+org.gradle.jvmargs=-Xmx2g
```

Comment:
this was required for me to be able to build the project the first time, but I can back it out if you want